test/yarp/errors_test.rb

```diff
@@ -923,7 +923,7 @@ def test_case_without_when_clauses_errors_on_else_clause
       )
 
       assert_errors expected, "case :a\nelse\nend", [
-        ["Unexpected `else` in `case` statement; a `when` clause must precede `else`", 8..12]
+        ["Expected a `when` or `in` clause after `case`", 0..4]
```

Comment:
Shouldn't we be adding a new test for the actual reported case in #1501, as well as updating the expectation of this test? Something like https://github.com/ruby/yarp/pull/1518/files#diff-7053a965faa5dfe2508b5e2e53dd765e7ec99bdfb2855555a10a1fae92157dd1R930-R943, for example.

Reply:
Updated!